get.go

```diff
@@ -0,0 +1,117 @@
+package gcfg
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+)
+
+// Get retrieves the values of a config field.
+func Get(config interface{}, field string) ([]string, error) {
```

Comment:
Not quite sure about `field` and the format it expects. It would break if there were unexpected `.` characters present.

Reply:
Yeah, I was thinking in that direction too. `section`, `subsection` and `variable` make sense to me since maps introduce another level (i.e. subsections).

I know what you mean, but I think this is fine since we define values as strings too, from a user-level/ini perspective. This is meant to return values as they were/would be set. I would have preferred to return a string for non-multi fields, but `[]string` also does the job.